ui/core/components/gear_picker/gear_picker.tsx

```diff
@@ -1088,20 +1047,27 @@ export class ItemList<T> {
 		this.equippedToItemFn = equippedToItemFn;
 		this.onItemClick = onItemClick;
 
-		const { slot, gearData } = config;
-		this.slot = slot;
+		this.slot = currentSlot;
 		this.gearData = gearData;
 		this.currentFilters = this.player.sim.getFilters();
 
 		const tabContentId = (label + '-tab').split(' ').join('');
-		const selected = label === config.selectedTab;
+		const selected = label === currentTab;
+
+		let itemLabel = 'Item';
```

Comment:
Maybe save these as enums like the `SelectorModalTabs`?

Reply:
Well these are `Item` for everything except for Reforge. Originally I had another case with more that I changed so probably doesn't need a switch stateent